sql/updates/pre-0.3.0--0.3.1-dev.sql

```diff
@@ -0,0 +1,25 @@
+DROP FUNCTION _timescaledb_internal.ddl_is_change_owner(pg_ddl_command);
+DROP FUNCTION _timescaledb_internal.ddl_change_owner_to(pg_ddl_command);
+
+DROP FUNCTION _timescaledb_internal.chunk_add_constraints(integer);
+DROP FUNCTION _timescaledb_internal.ddl_process_alter_table() CASCADE;
+
+CREATE INDEX ON _timescaledb_catalog.chunk_constraint(chunk_id, dimension_slice_id) WHERE dimension_slice_id IS NOT NULL;
+
+ALTER TABLE _timescaledb_catalog.chunk_constraint
+DROP CONSTRAINT chunk_constraint_pkey,
+ADD COLUMN constraint_name NAME;
+
+UPDATE _timescaledb_catalog.chunk_constraint
```

Comment:
Isn't it safer to read the constraint names for `pg_constraint` rather than assuming this name? That's what I do in my PR, so you can look at that query. What if someone manually changed the name of a constraint on a table? I don't think we block that in previous releases so it is entirely possible.